conda/base/context.py

```diff
@@ -949,11 +947,32 @@ def channels(self):
                 for rc_file in self.config_files
             )
             if argparse_channels and not channel_in_config_files:
+                deprecated.topic(
+                    "24.9",
+                    "25.3",
+                    topic=f"Adding '{DEFAULTS_CHANNEL_NAME}' to channel list implicitly.",
+                    addendum="To remove this warning, please choose a default channel explicitly "
+                    "via 'conda config --add channels <name>'. In the future, the implicit "
+                    "default channel configuration will be removed.",
+                    deprecation_type=FutureWarning,
```

Comment:
Let's move the addendum into a variable if it's repeated below

Reply:
how about we create a new property instead (and mark it for immediate deprecation) to solve this:

```python
    @property
    @deprecated(
        "24.9",
        "25.3",
        addendum=(
            "To remove this warning, please choose a default channel explicitly "
            "via 'conda config --add channels <name>'. In the future, the implicit "
            "default channel configuration will be removed."
        ),
        deprecation_type=FutureWarning,
    )
    def defaults(self) -> str:
        return DEFAULTS_CHANNEL_NAME
```

use this property instead of the `DEFAULTS_CHANNEL_NAME` within the `channels` property